boards/sparky_chibi/sparky_chibi.hpp

```diff
@@ -0,0 +1,484 @@
+
+
+/**
+ * \brief  Boardsupport for the Sparky board
+ *
+ */
+class Sparky_chibi
+{
+public:
+
+    static const uint8_t PWM_COUNT    = 10;
+
+    /**
+     * \brief   Configuration structure
+     */
+    struct conf_t
+    {
+        Gpio_chibios::conf_t      gpio_led_err;
+        Gpio_chibios::conf_t      gpio_led_stat;
+        Gpio_chibios::conf_t      gpio_led_rf;
+        Serial_chibios::conf_t    serial;
+        Pwm_chibios::conf_t       pwm[PWM_COUNT];
+        servo_conf_t              servo[PWM_COUNT];
+        I2c_chibios::conf_t       i2c1;
+        Barometer_MS5611::conf_t  barometer;
+    };
+
+
+
+    /**
+     * \brief           Constructor
+     *
+     * \param   config  Board configuration
+     */
+    Sparky_chibi(conf_t config = default_config());
+
+
+    /**
+     * \brief   Hardware initialisation
+
+     * \return  Success
+     */
+    bool init(void);
+
+
+    /**
+     * \brief   Default configuration for the board
+     *
+     * \return  Config structure
+     */
+    static inline conf_t default_config();
+
+
+    /**
+     * Public Members
+     */
+    // Gpio_stm32              led_err_gpio_;
+    // Gpio_stm32              led_stat_gpio_;
+    // Gpio_stm32              led_rf_gpio_;
+    // Led_gpio                led_err_;
+    // Led_gpio                led_stat_;
+    // Led_gpio                led_rf_;
+    // Pwm_stm32               pwm_0_;
+    // Pwm_stm32               pwm_1_;
+    // Pwm_stm32               pwm_2_;
+    // Pwm_stm32               pwm_3_;
+    // Pwm_stm32               pwm_4_;
+    // Pwm_stm32               pwm_5_;
+    // Pwm_dummy               pwm_6_;
+    // Pwm_dummy               pwm_7_;
+    // Serial_usb_stm32        serial_;
+    // Servo                   servo_0_;
+    // Servo                   servo_1_;
+    // Servo                   servo_2_;
+    // Servo                   servo_3_;
+    // Servo                   servo_4_;
+    // Servo                   servo_5_;
+    // Servo                   servo_6_;
+    // Servo                   servo_7_;
+    // Spi_stm32               spi_1_;
+    // Spi_stm32               spi_3_;
+    // I2c_stm32               i2c_1_;
+    // I2c_stm32               i2c_2_;
+    // State_display_sparky_v2 state_display_sparky_v2_;
+
+
+
+    Gpio_chibios            gpio_led_err_;
+    Gpio_chibios            gpio_led_stat_;
+    Gpio_chibios            gpio_led_rf_;
+    Led_gpio                led_err_;
+    Led_gpio                led_stat_;
+    Led_gpio                led_rf_;
+    Serial_chibios          serial_;
+    Pwm_chibios             pwm_[PWM_COUNT];
+    Servo                   servo_[PWM_COUNT];
+
+    State_display_sparky_v2 state_display_;
+
+    I2c_chibios             i2c1_;
+
+    Barometer_MS5611        barometer_;
+
+private:
+    /**
+     * /brief   Callback function that set PWM8 pin to high
+     *
+     * \detail  This is needed because PWM12 is not supported by CHibiOS, so we use PWM4 instead
```

Comment:
so, line 213
"so we use PWM4 instead" is wrong isn't it ?
should be "so we use PWM8 instead"